src/flow_solver/flow_solver_cases/periodic_turbulence.h

```diff
@@ -110,14 +110,23 @@ class PeriodicTurbulence : public PeriodicCubeFlow<dim,nstate>
     /// Function to compute the constant time step
     double get_constant_time_step(std::shared_ptr<DGBase<dim,double>> dg) const override;
 
-    /// Function to compute the adaptive time step
+/*    /// Function to compute the adaptive time step
```

Comment:
Could you delete these rather than commenting them?

Reply:
Deleted those lines :smile: